.github/workflows/release-minor.yml

```diff
@@ -20,11 +19,16 @@ jobs:
         with:
           github_token: ${{ secrets.GH_RW_TOKEN }}
           branch: refs/heads/master
+          bump-level: minor
```

Comment:
This is the only real change here. Everything else is prettier formatting.

Reply:
I'm confused? You're passing `bump-level` to https://github.com/ad-m/github-push-action which doesn't take a `bump-level` as an input... This will result in a warning from Github Actions stating the input wasn't expected. Or am I missing something?

Reply:
I'm going to test this more on my fork. 👍🏻